Documentation/Configuration/Typo3ConfVars/MAIL.rst

```diff
@@ -250,8 +251,8 @@ transport_smtp_stream_options
 .. confval:: $GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport_smtp_stream_options']
 
    :Path: $GLOBALS['TYPO3_CONF_VARS']['MAIL']
-   :type: bool
```

Comment:
I think both bool and array are allowed...?

Reply:
The only use of this setting in code I can see is this: $streamOptions = (array)($mailSettings['transport_smtp_stream_options'] ?? []);

"DefaultConfigurationDescription.yaml" uses "mixed" as type, "DefaultConfiguration.php" uses "null" as value, the unit test uses "array", the documentation here uses "bool" as type but the example is "array". So perhaps "mixed" would be ok but no other setting for the configuration in the documentation has "mixed" listed at the moment, but there are settings with "array"+"null".

Reply:
Ack